api-profiles/2017-03-09-profile.json

```diff
@@ -0,0 +1,48 @@
+{
+    "resource-manager": {
+        "microsoft.authorization": {
+            "version": "2015-07-01",
+            "types": {
+                "locks":"2015-01-01",
+                "policyassignments": "2015-10-01-preview",
+                "policydefinitions": "2015-10-01-preview"
+            }
+        },
+        "microsoft.compute": {
+            "version": "2016-03-30",
+            "types": {}
+        },
+        "microsoft.keyvault": {
+            "version": "2016-10-01",
+            "types": {}
+        },
+        "microsoft.network": {
+            "version": "2015-06-15",
+            "types": {}
+        },
+        "microsoft.storage": {
+            "version": "2016-01-01",
+            "types": {}
+        },
+        "microsoft.subscription": {
+            "version": "2016-02-01"
```

Comment:
there is subscription section under `microsoft.resources` with the different API version. Aren't they gona conflict?

Reply:
Its a mistake. I am fixing. Subscriptions is not a resource type under resources namespace. Going to remove from here. Thanks for taking a look. I am going to send a mail after adding a readme.md on the structure